tools/packaging/scripts/configure-hypervisor.sh

```diff
@@ -435,6 +435,9 @@ generate_qemu_options() {
 	# Support Ceph RADOS Block Device (RBD)
 	[ -z "${static}" ] && qemu_options+=(functionality:--enable-rbd)
 
+	# Support NUMA topology
+	[ "${NUMA_ENABLED}" = "yes" ] && qemu_options+=(functionality:--enable-numa)
```

Comment:
The SC job reports two issues here :
- Prefer [[ ]] over [ ] for tests in Bash/Ksh. [SC2292]
- warning: NUMA_ENABLED is referenced but not assigned. [SC2154]